src/index.html

```diff
@@ -1,12 +1,287 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="page__header header">
+      <a href="#" class="header__logo">
+        <img src="moyo.jpg" alt="Logo"
+         class="header__logo-image">
+      </a>
+      <nav>
+        <ul class="header__menu">
+          <li class="header__menu-item"><a class="header__menu-link header__menu-link--is-active" href="#">apple</a></li>
+          <li class="header__menu-item"><a class="header__menu-link" href="#">samsung</a></li>
+          <li class="header__menu-item"><a class="header__menu-link" href="#">smartphones</a></li>
+          <li class="header__menu-item">
+            <a class="header__menu-link header__menu-link--hover" href="#"
+            data-qa="nav-hover">laptops & computers</a>
+          </li>
+          <li class="header__menu-item"><a class="header__menu-link" href="#">gadgets</a></li>
+          <li class="header__menu-item"><a class="header__menu-link" href="#">tablets</a></li>
+          <li class="header__menu-item"><a class="header__menu-link" href="#">photo</a></li>
+          <li class="header__menu-item"><a class="header__menu-link" href="#">video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <article class="card" data-qa="card">
+          <div class="card__img-container">
+            <img src="./images/imac.jpeg"
+               class="card__img"
+              alt="picture"
+            >
+          </div>
+          <p class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+
+          <button class="card__button" data-qa="card-hover">BUY</button>
+        </article>
+        <article class="card" data-qa="card">
```

Comment:
Add the blank line between the cards
```suggestion
        </article>
        
        <article class="card" data-qa="card">
```